test/github-issues/9049/entity/post.entity.ts

```diff
@@ -0,0 +1,14 @@
+import { ObjectID } from "../../../../src/driver/mongodb/typings"
+import { Type } from "class-transformer"
```

Comment:
is it required? Please remove it if not, we want to keep tests simple as possible

Reply:
removed. Thanks